configs/override_rules.yaml

```diff
@@ -0,0 +1,17 @@
+---
+
+# Examples:
+#server_group:
+#- when:
+#    id:
+#    - 14ba292e-f0bb-4eec-9b3a-abb775db7ff4
+#    - 533d5840-8584-47bc-be16-2825536e9027
+#    - 7398b91f-e1c2-4378-9943-391318981aee
+#    - 1da4d59e-06de-4d34-b0e7-655e886e0183
+#  replace: 66e76c27-874d-48e1-bca2-b0eb0ea95ff3
+
+#availability_zone:
+#- when: us_west_dc1_ssd
+#  replace: us_west_dc2_ssd
+#- when: null
+#  replace: us_west_dc2
```

Comment:
According to the code it also allows to write rules for flavors and keys. Let's have examples of those here too.